workflow/snakemake_rules/common.smk

```diff
@@ -164,18 +164,20 @@ def _get_upload_inputs(wildcards):
     # mapping of remote → local filenames
     uploads = {
         f"aligned.fasta.xz":              f"results/aligned_{origin}.fasta.xz",              # from `rule align`
-        f"sequence-diagnostics.tsv.xz":   f"results/sequence-diagnostics_{origin}.tsv.xz",   # from `rule diagnostic`
-        f"flagged-sequences.tsv.xz":      f"results/flagged-sequences_{origin}.tsv.xz",      # from `rule diagnostic`
-        f"to-exclude.txt.xz":             f"results/to-exclude_{origin}.txt.xz",             # from `rule diagnostic`
         f"masked.fasta.xz":               f"results/masked_{origin}.fasta.xz",               # from `rule mask`
         f"filtered.fasta.xz":             f"results/filtered_{origin}.fasta.xz",             # from `rule filter`
         f"mutation-summary.tsv.xz":       f"results/mutation_summary_{origin}.tsv.xz",       # from `rule mutation_summary`
     }
 
     for build_name in config["builds"]:
         uploads.update({
-            f"{build_name}/sequences.fasta.xz": f"results/{build_name}/{build_name}_subsampled_sequences.fasta.xz",
-            f"{build_name}/metadata.tsv.xz":    f"results/{build_name}/{build_name}_subsampled_metadata.tsv.xz",
+            f"{build_name}/sequences.fasta.xz": f"results/{build_name}/{build_name}_subsampled_sequences.fasta.xz",   # from `rule combine_samples`
+            f"{build_name}/metadata.tsv.xz":    f"results/{build_name}/{build_name}_subsampled_metadata.tsv.xz",      # from `rule combine_samples`
+            f"{build_name}/aligned.fasta":      f"results/{build_name}/aligned.fasta",                                # from `rule build_align`
+            # export the auspice dataset which matches the subsampled sequences / metadata (see `rule finalize`)
+            f"{build_name}/{config['auspice_json_prefix']}_{build_name}.json":                 f"auspice/{config['auspice_json_prefix']}_{build_name}.json",
+            f"{build_name}/{config['auspice_json_prefix']}_{build_name}_tip-frequencies.json": f"auspice/{config['auspice_json_prefix']}_{build_name}_tip-frequencies.json",
+            f"{build_name}/{config['auspice_json_prefix']}_{build_name}_root-sequence.json":   f"auspice/{config['auspice_json_prefix']}_{build_name}_root-sequence.json"
```

Comment:
This will produce paths like:

    https://data.nextstrain.org/files/ncov/open/global/ncov_open_global.json
    https://data.nextstrain.org/files/ncov/open/global/ncov_open_global_tip-frequencies.json

What about instead producing paths like this:

    https://data.nextstrain.org/files/ncov/open/global.json
    https://data.nextstrain.org/files/ncov/open/global_tip-frequencies.json

with this mapping?

```python
            f"{build_name}.json":                 f"auspice/{config['auspice_json_prefix']}_{build_name}.json",
            f"{build_name}_tip-frequencies.json": f"auspice/{config['auspice_json_prefix']}_{build_name}_tip-frequencies.json",
            f"{build_name}_root-sequence.json":   f"auspice/{config['auspice_json_prefix']}_{build_name}_root-sequence.json"
```

Reply:
Seems sensible - updated this PR with those recommendations.

Reply:
Thanks! Though it looks like the change you made wasn't quite what I meant. With the latest changes on this PR, the URLs are now:

```
https://data.nextstrain.org/files/ncov/open/global/global.json
https://data.nextstrain.org/files/ncov/open/global/global_tip-frequencies.json
```

but I meant URLs like:

```
https://data.nextstrain.org/files/ncov/open/global.json
https://data.nextstrain.org/files/ncov/open/global_tip-frequencies.json
```

Does that seem reasonable?

Reply:
It's a judgement call I think - both seem reasonable to me. I like the fact that the auspice JSONs are co-located with the intermediates from which they are generated.

Reply:
Nod. Ah, I like the fact that as `data.nextstrain.org/files/ncov/open/global.json` it more closely matches the `nextstrain.org/ncov/open/global` URL. I'm ok with either way, though. Do others have input? @trvrb?